eclass/toolchain-funcs.eclass

```diff
@@ -5,13 +5,19 @@
 # @MAINTAINER:
 # Toolchain Ninjas <toolchain@gentoo.org>
 # @BLURB: functions to query common info about the toolchain
+# @SUPPORTED_EAPIS: 5 6 7
 # @DESCRIPTION:
 # The toolchain-funcs aims to provide a complete suite of functions
 # for gleaning useful information about the toolchain and to simplify
 # ugly things like cross-compiling and multilib.  All of this is done
 # in such a way that you can rely on the function always returning
 # something sane.
 
+case ${EAPI:-0} in
+	[567]) ;;
```

Comment:
Now, this is going to block mah python-r1 EAPI 8 support :-(.

Reply:
Should be fixed once PR #21355 is merged.